src/options/smt_options.toml

```diff
@@ -9,6 +9,66 @@ name   = "SMT Layer"
   default    = "false"
   help       = "eliminate functions by ackermannization"
 
+[[option]]
```

Comment:
Would it make sense to put these options into a separate module? I'm worried about smt_options becoming too cluttered.  @nafur

Reply:
I think so, yes.
Both for the user and within the implementation, putting everything related to parallelization in its own module is probably a good idea.

@amaleewilson: Simply put the new options in their own file (`src/options/parallel_options.toml`) and add this file to https://github.com/cvc5/cvc5/blob/master/src/CMakeLists.txt#L1189.

Reply:
I think this would be a good idea because there will most likely be more.